deployment/clouddeploy/gke-workers/base/nvd-cve-osv.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: nvd-cve-osv
+spec:
+  timeZone: Australia/Sydney
+  schedule: "0 6 * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      activeDeadlineSeconds: 86400
+      template:
+        spec:
+          containers:
+          - name: nvd-cve-osv
+            image: nvd-cve-osv
```

Comment:
Going to need to build this image in Cloud Build like here:
https://github.com/google/osv.dev/blob/5433f3c427dd76c4618c084389107ac6140ab0c2/deployment/build-and-stage.yaml#L151-L159

Reply:
Thanks, added in [90d0746](https://github.com/google/osv.dev/pull/1306/commits/90d07468737e2b1c4963f0fff12dfec3288c063a)